### PR TITLE
XS✔ ◾ Update rule.md

### PR DESCRIPTION
**Pull request by ae-cher-ssw**
Rule cleaning per Levi request 

1. Fixing subheading
2. adding image
3. fixing main heading

### CODE REVIEW

File: rules/connect-client-im/rule.md [hunk 1]
```diff
@@ -13,18 +13,22 @@ guid: 4332723e-c780-4d0c-a759-5067404ed8fc
 
 Effective communication with your clients on Microsoft Teams builds credibility, trust and ensures they receive the deserved attention.
 
+![Figure: Keeping Teams at the centre of client communication](Teams image)
+
 <!--endintro-->
 
-# Do you know why it’s important to talk with clients on Teams? 
+# Talking to clients on Teams
```
Comment (JackDevAU) [inline rules/connect-client-im/rule.md:20]: I feel the only h1 (#) should be the title, I think a h2 here is a good replacement

From: 

# Talking to clients on Teams

To: 

## Talking to clients on Teams

**Review by JackDevAU — APPROVED** [review]

File: rules/connect-client-im/rule.md (hunk 1 above)
Comment (JackDevAU) [inline rules/connect-client-im/rule.md:20]:
```suggestion
## Talking to clients on Teams
```

**Review by JackDevAU — APPROVED** [review]

**Comment by tiagov8** [comment]
Hi @JackDevAU @ae-cher-ssw 

❌ This rule went up with a broken image!

1. I added a task here, but it was not actioned:

Screenshot 2024-01-08 at 9 29 15 AM

**Comment by JackDevAU** [comment]
Hey @tiagov8 

I can't see those comments?! 😮 Did it submit your review?

**Comment by tiagov8** [comment]
Pasting my comments here:

Hi @ae-cher-ssw

**Line 16** - The file name for the image here is wrong. This would create a broken image.

 1. Please update the file name path accordingly
 
---

**Line 18** - We don't use H1 in rules content. It's a good practice to only have one H1 on a page, and that H1 is already used in the rule title.

 1. Please change to be a H2
 
 ---

**Line 47** - This caption seems unrelated to the greybox. I think it should say "Figure: Letting the client know they can reach you via IM as an option"

 1. Please action if agreed

---

**Line 28** and others

1. Can you find and fix all occurances of "Organisation(s)" to follow https://www.ssw.com.au/rules/spelling-do-you-use-us-english

**Pull request merged**